src/DssSpell.sol

```diff
@@ -25,14 +35,29 @@ contract SpellAction {
     //  against the current release list at:
     //     https://changelog.makerdao.com/releases/mainnet/1.1.2/contracts.json
 
-    //address constant MCD_JUG         = 0x19c0976f590D67707E62397C87829d896Dc0f1F1;
+    address constant MCD_VAT         = 0x35D1b3F3D7966A1DFe207aa4514C12a259A0492B;
+    address constant MCD_CAT         = 0xa5679C04fc3d9d8b0AaB1F0ab83555b301cA70Ea;
+    address constant MCD_JUG         = 0x19c0976f590D67707E62397C87829d896Dc0f1F1;
+    address constant MCD_SPOT        = 0x65C79fcB50Ca1594B025960e539eD7A9a6D434A3;
+    address constant MCD_POT         = 0x197E90f9FAD81970bA7976f33CbD77088E5D7cf7;
+    address constant MCD_END         = 0xaB14d3CE3F733CACB76eC2AbE7d2fcb00c99F3d5;
+    address constant FLIPPER_MOM     = 0xc4bE7F74Ee3743bDEd8E0fA218ee5cf06397f472;
+    address constant OSM_MOM         = 0x76416A4d5190d071bfed309861527431304aA14f;
+    address constant ILK_REGISTRY    = 0x8b4ce5DCbb01e0e1f0521cd8dCfb31B308E52c24;
+    address constant CHAINLOG        = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
+
+    // ETH-B specific addresses
+    address constant ETH            = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
```

Comment:
Nit: could we name this `WETH` instead of `ETH`?

Reply:
We have been sticking with changelog keys here, we use ETH there.